src/components/Layer2NetworksTable/hooks/useNetworkColumns.tsx

```diff
@@ -0,0 +1,241 @@
+"use client"
+
+import { IoChevronDownSharp, IoChevronUpSharp } from "react-icons/io5"
+import { MdInfoOutline } from "react-icons/md"
+import { ColumnDef } from "@tanstack/react-table"
+
+import { ExtendedRollup, Lang } from "@/lib/types"
+
+import { TableMeta } from "@/components/DataTable"
+import { TwImage } from "@/components/Image"
+import NetworkMaturityTooltip from "@/components/Layer2NetworksTable/NetworkMaturityTooltip"
+import InlineLink from "@/components/Link"
+import Tooltip from "@/components/Tooltip"
+import { TableCell, TableHead } from "@/components/ui/Table"
+
+export const useNetworkColumns: ColumnDef<ExtendedRollup>[] = [
+  {
+    id: "l2Info",
+    header: ({ table }) => {
+      return (
+        <TableHead className="flex-1">
+          <p>
+            Networks showing <strong>({table.options.data.length})</strong>
+          </p>
+        </TableHead>
+      )
+    },
+    cell: ({ table, row }) => {
+      const meta = table.options.meta as TableMeta
+      return (
+        <TableCell
+          className={`flex flex-1 flex-row items-center justify-between lg:table-cell ${row.original.cantExpand ? "border-b-4" : ""}`}
```

Comment:
Can use `cn` here to merge these
```suggestion
          className={cn(
            "flex flex-1 flex-row items-center justify-between lg:table-cell",
            row.original.cantExpand && "border-b-4"
          )}
```